NewStarterProject/Program.cs

```diff
@@ -6,9 +10,13 @@
 
 builder.Services.AddControllers();
 
+builder.Services.AddDbContext<StarterProjectContext>(options =>
+    options.UseSqlServer(builder.Configuration.GetConnectionString("DatabaseConnection")));
```

Comment:
For a new starter project this is not something to worry about, but for P10 its important to keep in mind that ideally connection strings and secrets should be passed into the application using environment variables. (you probably know this already haha)

The reason I am making a point here is that unfortunately we have a few places where secrets and connection strings might be hard coded into the codebase for P10 but we should try and steer away from doing so going forward. 

Like I said for a new starter project his is not an issue and completely find but I just wanted to make you aware when moving to coding on P10 👍